content/en/docs/user-guides/cli-reference/notation_verify.md

```diff
@@ -10,40 +12,37 @@ Verify OCI artifacts
 
 Prerequisite: added a certificate into trust store and created a trust policy.
 
-Example - Verify a signature on an OCI artifact identified by a digest:
-```shell
-notation verify <registry>/<repository>@<digest>
 ```
-
-Example - Verify a signature on an OCI artifact identified by a tag  (Notation will resolve tag to digest):
-```shell
-notation verify <registry>/<repository>:<tag>
+notation verify [reference] [flags]
 ```
 
-Example - [Experimental] Verify a signature on an OCI artifact referenced in an OCI layout using trust policy statement specified by scope.
-```shell
-notation verify --oci-layout <registry>/<repository>@<digest> --scope <trust_policy_scope>
-```
+### Examples
 
-Example - [Experimental] Verify a signature on an OCI artifact identified by a tag and referenced in an OCI layout using trust policy statement specified by scope.
-```shell
-notation verify --oci-layout <registry>/<repository>:<tag> --scope <trust_policy_scope>
 ```
```

Comment:
@zr-msft  For experimental features, is there a way to show the examples in CLI reference docs? It seems experimental flags and their examples are not generated by Cobra.